packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -21,3 +21,14 @@ export const sync = () => ({
       'Apply synchronization when there is a 100% match between organization and snapshot resources.',
   }),
 });
+
+export const previewLevel = () => ({
+  previewLevel: flags.enum({
+    char: 'p',
+    description:
+      'Whether to only display a light or detailed preview of the overall changes',
```

Comment:
I would add some explanation about why someone might want to use a `light` preview instead of always a `detailed` one.

Something that explains why it can save time.